lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
 // lms - views - shopping cart
 // ====================
 
+$enrollment-details-width-difference: $baseline*11.85;
+$data-input-width-difference: $baseline*12.25;
+$date-span-width-difference: $baseline*35;
+$cart-list-border: 1px solid $border-color-1;
+$hr-border: 1px solid $dark-gray2;
+$input-border: 2px solid $dark-gray2;
+$discount-border: 2px solid $light-gray1;
+$steps-border: 3px solid $light-gray1;
+$light-border: 1px solid $gray-l5;
+$expiry-border-color: #F0F0F0;
```

Comment:
Please avoid using hex-values directly. When possible, use the colors defined in _variables.scss

Additionally, try to avoid using "Old Variables" and "edx.org marketing site variables." When possible, I would use the base colors section of that page as well.